packaging/db/zip/linux/ignite3db

```diff
@@ -59,7 +59,19 @@ start() {
 }
 
 stop() {
-  kill -9 "$(cat ${IGNITE_HOME}/pid)"
+  kill -15 "$(cat ${IGNITE_HOME}/pid)"
+
+  max_attempts=10
+  count=0
+  while [ps -p "$(cat ${IGNITE_HOME}/pid)" > /dev/null]; do
+    sleep 10
+    count=$((count + 1))
+  done
+
+  if [ ps -p "$(cat ${IGNITE_HOME}/pid)" > /dev/null ]; then
```

Comment:
Use 'if cmd; then ..' to check exit code, or 'if [[ $(cmd) == .. ]]' to check output